src/plugins/intel_gna/tests/functional/subgraph_tests/input_conv_add_transpose.cpp

```diff
@@ -0,0 +1,103 @@
+                   std::vector<size_t>>                 // Input Shape
+    InputConvAddParams;
+
+namespace LayerTestsDefinitions {
+
+class InputConvAddTransposing : public testing::WithParamInterface<InputConvAddParams>,
+                                public LayerTestsUtils::LayerTestsCommon {
+public:
+    static std::string getTestCaseName(testing::TestParamInfo<InputConvAddParams> obj) {
+        InferenceEngine::Precision precision;
+        std::string targetDevice;
+        std::map<std::string, std::string> configuration;
+        std::vector<size_t> input_shape;
+
+        std::tie(precision, targetDevice, configuration, input_shape) = obj.param;
+
+        std::ostringstream result;
+        result << "netPRC=" << precision.name() << "_";
+        result << "targetDevice=" << targetDevice << "_";
+
+        for (auto const& configItem : configuration) {
+            result << "_configItem=" << configItem.first << "_" << configItem.second;
+        }
+        result << "_inputShape=" << CommonTestUtils::vec2str(input_shape);
+
+        return result.str();
+    }
+
+protected:
+    void SetUp() override {
+        InferenceEngine::Precision precision;
+        std::vector<size_t> input_shape;
+        std::tie(precision, targetDevice, configuration, input_shape) = this->GetParam();
+
+        auto ng_precision = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(precision);
+        auto input = std::make_shared<Parameter>(ng_precision, ov::Shape{input_shape});
+
+        const std::vector<size_t> filter_size{1, 1};
+        const std::vector<size_t> strides{1, 1};
+        const std::vector<ptrdiff_t> pads_begin{0, 0};
+        const std::vector<ptrdiff_t> pads_end{0, 0};
+        const std::vector<size_t> dilations{1, 1};
+        const auto pad_type = ov::op::PadType::EXPLICIT;
+        const size_t out_channels_num = 8;
+        constexpr auto c_index_in_nchw = 1;
+        const auto weights_size = ov::shape_size(filter_size) * out_channels_num * input_shape[c_index_in_nchw];
+        auto weights_values = CommonTestUtils::generate_float_numbers(weights_size, -0.2f, 0.2f);
+
+        auto convolution = ngraph::builder::makeConvolution(input,
+                                                            ng_precision,
+                                                            filter_size,
+                                                            strides,
+                                                            pads_begin,
+                                                            pads_end,
+                                                            dilations,
+                                                            pad_type,
+                                                            out_channels_num,
+                                                            false,
+                                                            weights_values);
+        auto add = std::make_shared<Add>(input, convolution);
```

Comment:
It makes sense to add tests where `Parameter` is second input. And the same for `Const` to cover all use cases.

Reply:
fixed